lib/util.js

```diff
@@ -403,6 +403,108 @@ module.exports = {
     };
   },
 
+  /**
+   * Generates object containing objects of tags and their requests
+   * @param {object} spec - openAPI spec object
+   * @returns {object} returns an object containing objects of tags and their requests
+   */
+  /* eslint-disable max-depth */
```

Comment:
why is this eslint disable needed at a function level?

Reply:
Actually there were a few max-depth disable line in the function..so i thought to add this at the starting of the function.

Reply:
no - we shouldn't be using this so liberally. We should revisit each violation and check how it can be addressed.